.github/workflows/ci.yml

```
@@ -2151,7 +2156,7 @@ jobs:
           echo "✅ Homepage smoke passed"
 
   deploy:
-    needs: [ci-fast, ci-build, ci-unit-tests, ci-e2e-tests, ci-smoke-required, ci-homepage-smoke] # Required smoke + homepage smoke must pass before deploy
+    needs: [ci-fast, ci-build, ci-unit-tests, ci-e2e-tests, ci-smoke-required] # Required smoke must pass before deploy; homepage smoke is informational
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)